package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "data-api-image",
+  "version": "0.0.1",
+  "description": "Instantiation of data-api along with its respective build and deploy scripts reside here",
+  "dependencies": {
+    "express": "^4.16.2",
+    "validator": "^9.2.0",
+    "hypothesis-core-image": "github:fabric8-hdd/hypothesis-core-image",
+    "data-api": "github:fabric8-hdd/data-api"
+  },
+  "scripts": {
+    "start": "node main.js",
+    "test": "mocha --timeout 5000 tests/*_test.js"
```

Comment:
latest versions of mocha don't return the control after running tests.
So you might have to append  --exit at the end like in https://github.com/fabric8-hdd/manager-api-image/blob/master/package.json#L8